modules/components/pages/inputs/schema_registry.adoc

```diff
@@ -0,0 +1,285 @@
+= schema_registry
+:type: input
+:status: beta
+:categories: ["Integration"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Reads schemas from a schema registry.
```

Comment:
@Jeffail. Is there a specific use case for this connector? Can we describe it hear? 

Also, is this a community or certified connector? I have assumed certified.

Reply:
It's intended to allow users to extract all of the schemas from a registry in order to back them up elsewhere.

Reply:
@Jeffail. Updated description. Please could you check it is accurate?